src/group_theory/subgroup/zpowers.lean

```diff
@@ -29,14 +29,13 @@ subgroup.copy (zpowers_hom G g).range (set.range ((^) g : ℤ → G)) rfl
 
 @[simp] lemma mem_zpowers (g : G) : g ∈ zpowers g := ⟨1, zpow_one _⟩
 
+lemma _root_.set.range_zpow (g : G) : set.range (λ n : ℤ, g ^ n) = zpowers g := rfl
+
```

Comment:
This looks backwards to me, I'd expect
```suggestion
lemma coe_zpowers (g : G) : \u(zpowers g) = set.range (λ n : ℤ, g ^ n) := rfl

```
(complete with the coe I'm to lazy to type

Reply:
Do you want it `norm_cast` as well?

Reply:
Sure, why not